docs/docusaurus/docs/reference/learn/data_quality_use_cases/uniqueness.md

```diff
@@ -0,0 +1,263 @@
+  ```
+
+3. Add an **Expect column unique value count to be between** Expectation to the freshly created Data Asset.
+4. Populate the Expectation:
+   * Select `transfer_type` as the **Column**.
+   * Provide a **Min Value** of `2` and a **Max Value** of `4`.
+5. Save the Expectation.
+6. Click the **Validate** button.
+7. Review Validation Results.
+</TabItem>
+
+<TabItem value="gx_core" label="GX Core">
+Run the following GX Core workflow.
+
+```python title="" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/uniqueness_resources/uniqueness_workflow.py full example code"
+```
+</TabItem>
+
+</Tabs>
+
+**GX solution**: GX enables validating the count of unique values in a column. By setting a min and max value, you can ensure that the number of distinct values falls within an expected range. This can be done using either GX Core or GX Cloud.
+
+In this example, we expect the `transfer_type` column to have between 2 and 4 unique values. The `ExpectColumnUniqueValueCountToBeBetween` Expectation allows us to codify this requirement and validate it against our data. If new transfer types are introduced or if data errors lead to unexpected values, the validation will fail, alerting us to potential data quality issues.
+
+## Scenarios
+
+### Detecting duplicate transactions
+
+**Context**: In financial systems, duplicate transactions can lead to incorrect account balances, unhappy customers, and accounting discrepancies. Monitoring key fields for uniqueness helps detect and prevent such issues.
+
+**GX solution**: Use `ExpectCompoundColumnsToBeUnique` to validate that the combination of fields that uniquely identify a transaction (e.g., timestamp, sender account, recipient account, amount) is unique across all rows.
+
+### Ensuring integrity of customer records
+
+**Context**: In a customer database, each customer should have a unique identifier. Duplicate customer IDs can lead to severe data integrity issues, such as incorrectly merged customer profiles, misdirected communications, or inaccurate analytics. If not caught early, resolving duplicate records can become a complex, error-prone, and resource-intensive process.
+
+**GX solution**: Use `ExpectColumnValuesToBeUnique` to ensure that the customer ID column contains only unique values. If duplicates are found, investigate and resolve them to maintain data integrity.
+
+### Validating allowed payment types
+
+**Context**: In an e-commerce system, the payment type field should only contain valid, predefined values (e.g., "credit_card", "debit_card", "paypal"). Unexpected payment types could indicate data entry errors or system issues.
+
+**GX solution**: Use `ExpectColumnDistinctValuesToEqualSet` to validate that the distinct values in the payment type column exactly match the set of allowed payment types. If unexpected values are found, investigate and resolve the discrepancy.
+
+### Monitoring for missing sensor readings
```

Comment:
How does this scenario correspond to uniqueness?

Reply:
removed 1195d9aee259a1198ed7ccc0db2e406830c9e3ec